### ISSUE TITLE
Loss fluctuations

### ISSUE THREAD

**Issue by Wilsker** [Wilsker/tdsm_encoder#2]
Current model has a fluctuating loss after only a few epochs. This could be a problem with the loss calculation for plots. Could also be something odd in the training as it seems to learn very quickly and then stop.
![loss_v_epoch](https://user-images.githubusercontent.com/5683742/221140376-560196f6-43ca-44a0-b2c3-e031a37218c3.png)

**Comment by Wilsker** [comment]
Loss wasn't calculated correctly. Too large learning rate also causing non-convergence.
